### PR TITLE
curiefense using the new PR format for rancher/partner-charts1

### PR DESCRIPTION
**Pull request by JuanPabloGarciaMonzon**
curiefense using the new PR format for rancher/partner-charts1

### CODE REVIEW

**Comment by PennyScissors** [comment]
Hello @JuanPabloGarciaMonzon, can you please refactor this PR into two separate commits as following to make it easier to review?

1st commit: This commit should contain your changes done in `package/` to add your chart (Basically what you did [here](https://github.com/rancher/partner-charts/pull/107/commits/62d5b353f57d7d9e16d24f9bb858026b436887cf))

2nd commit: Run `make charts` and add everything that gets updated to a second commit (Usually `assets`, `charts` and in some cases `index.yaml` as well).

Then run `make validate` locally and if it passes it should be ready for review.

File: packages/curiefense/generated-changes/patch/Chart.yaml.patch [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
++++ charts/Chart.yaml
+@@ -2,4 +2,10 @@
+ appVersion: 1.0.0
+ description: Complete curiefense deployment
+ name: curiefense
+-version: 1.0.0
+\ No newline at end of file
++version: 1.0.0
++
++kubeVersion: # A SemVer range of compatible Kubernetes versions. E.g 1.18 - 1.21, >= 1.19, etc
```
Comment (PennyScissors) [inline packages/curiefense/generated-changes/patch/Chart.yaml.patch:11]: You need to set the value of `kubeVersion` to a semver range of K8s versions that are compatible with your chart. For example, if your charts is compatible with k8s v1.18 and above you would set this to `kubeVersion: ">= 1.18"`.

File: packages/curiefense/generated-changes/patch/Chart.yaml.patch [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
++annotations:
++  catalog.cattle.io/certified: partner # Enables the "partner" badge in the UI for easier identification
++  catalog.cattle.io/release-name: curiefense # Your chart's name in kebab-case, this is used for deployment
++  catalog.cattle.io/display-name: curiefense # The chart's name you want displayed in the UI
```
Comment (PennyScissors) [inline packages/curiefense/generated-changes/patch/Chart.yaml.patch:15]: Please remove the comments, they are not necessary.

**Comment by JuanPabloGarciaMonzon** [comment]
So i need to close this pull request and upload two new ones, one with the
part 1 and other with the part2?

**Comment by JuanPabloGarciaMonzon** [comment]
So i need to close this pull request and upload two new ones, one with the
part 1 and other with the part2?

**Comment by PennyScissors** [comment]
No, just one PR with 2 commits. You can force push to this one to overwrite your current commits, or submit a new PR, whatever is easier for you.

**Comment by JuanPabloGarciaMonzon** [comment]
Hi Steven i have accomplished the requirements of the two commits in one PR
and the command "make validate" has been successful.
I made a new PR i deleted the old PR.
[image: image.png]
Cheers and thank you for your  help.
Here are the photos of the make validate.
[image: image.png]
[image: image.png]
[image: image.png]

**Pull request closed without merging**